stdlib/universe/moving_average.go

```diff
@@ -237,9 +231,6 @@ func (t *movingAverageTransformation) passThroughBool(ts *array.Int64, vs *array
 	l := vs.Len()
 	for ; i < l; i++ {
 		cTime := execute.Time(ts.Value(i))
-		if cTime < pTime {
```

Comment:
Could you please remove every check on time? Also below.

Reply:
I think this is an outdated file